web/styles/left_sidebar.css

```diff
@@ -635,7 +635,6 @@ li.active-sub-filter {
        the order of home views, using a single
```

Comment:
I think more can be removed here. See the comment about "We display this as grid...."